app/src/processing/app/ui/theme/m3/Theme.kt

```diff
@@ -0,0 +1,301 @@
+
+
+import androidx.compose.material3.lightColorScheme
+import androidx.compose.material3.darkColorScheme
+import androidx.compose.runtime.Immutable
+import androidx.compose.ui.graphics.Color
+
+@Immutable
+data class ExtendedColorScheme(
+    val warning: ColorFamily,
+)
+
+val lightScheme = lightColorScheme(
```

Comment:
Are all the color values being used once in these schemes? Maybe the `Color.kt` stuff isn't necessary and we can just define the values here without adding indirection through the variable declarations?